FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Beniamin Idziak on 15/08/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not intended to be subclassed, so it can be final:

```suggestion
final class ManagedFeedImage: NSManagedObject {
```